recipes/hdf5/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: hdf5
+  version: 1.8.16
```

Comment:
Could try and replace versions using jinja templating to keep the maintenance effort light. Here's an [example]( https://github.com/conda-forge/webcolors-feedstock/blob/master/recipe/meta.yaml ) of something we have been trying to do.